### DISCUSSION TITLE
有单独的TRT版MEMC_Rife_NV插件下载吗？或请考虑在下一版懒人包集成进去？

### DISCUSSION THREAD

**Discussion by ligs85** [hooke007/mpv_PlayKit#393] [mpv 通用]
如题，相信很多人会为了AI插帧功能而选择此项目吧？
而且使用nVidia 20系以上显卡的用户应该也是主流，希望能单独提供MEMC_Rife_NV插件下载，否则为此必须要下载庞大的Mega包... 
当然更希望能在下一班懒人包中继承NV专版，谢谢！

**Comment by hooke007** [comment]
你看过说明就不会这么问了。
你可以看看光一个vsmlrt多大
